src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -571,7 +571,28 @@ def transform_manifest(self, format: str, filters: JSON):
         if format == 'full':
             source_filter = ['contents.metadata.*']
             entity_type = 'bundles'
-            manifest_config = self.generate_metadata_manifest()
+            es_search = self.create_request(filters,
+                                            self.es_client,
+                                            request_config,
+                                            post_filter=False,
+                                            source_filter=source_filter,
+                                            enable_aggregation=False,
+                                            entity_type=entity_type)
+            es_search.aggs.metric('fields', 'scripted_metric',
+                                  init_script='params._agg.fields = new HashSet()',
+                                  map_script='for (row in params._source.contents.metadata) { '
```

Comment:
Triple quoted string with newlines and indented Painless code would be better. And extract into separate variable.